sql/dd/impl/types/object_table_impl.h

```diff
@@ -125,6 +127,8 @@ class Object_table_impl : virtual public Object_table {
 
   virtual void set_encrypted();
```

Comment:
may be rename this set_target_encrypted() instead of generic set_encrypted(). We have sibling set_actual_encrypted().